src/app/map/support/map-utils.ts

```diff
@@ -188,9 +188,9 @@ function queryLayer(featureLayer: __esri.FeatureLayer, params: __esri.Query): Pr
 // Warning! Esri promises are not compatible with native JS Promises
 // Adding `done` handler avoids infinite loop in the browser...
 // (see https://www.esri.com/arcgis-blog/products/js-api-arcgis/mapping/making-better-promises)
-function whenReady(esriLoadable: IPromise<any>): IPromise<any> {
-  return esriLoadable.then(done, fail);
-}
+// function whenReady(esriLoadable: IPromise<any>): IPromise<any> {
```

Comment:
To remove or being fixed?

Reply:
Being fixed. We'll need to update our esriLoader and get rid of the deprecated API calls. Not important for right now... not sure if we're sticking with the ESRI js API or dumping it for leaflet. If we're dumping it for leaflet, all this code will be removed anyway